client/src/components/SignUp/AmplifySignIn.js

```diff
@@ -9,6 +9,7 @@ import userState from '../../utils/UserState';
 import { useRecoilState } from 'recoil';
 import { Redirect } from 'react-router';
 import AmplifyGoogle from './AmplifyGoogle';
+import LoginGoogle from '../Login/LoginGoogle';
```

Comment:
Oops -- forgot I added this line. It can be removed - I added it while trying to figure out how the stupid button was getting loaded onto the page in the first place, and this was NOT how haha